google/resumable_media/_helpers.py

```diff
@@ -157,7 +158,8 @@ def wait_and_retry(func, get_status_code, retry_strategy):
     """
     total_sleep = 0.0
     num_retries = 0
-    base_wait = 0.5  # When doubled will give 1.0
+    # base_wait will be multiplied by the multiplier on the first retry.
+    base_wait = float(retry_strategy.initial_delay) / retry_strategy.multiplier
```

Comment:
This change needs coverage in `tests/unit/test__helpers.py` (presumably by setting non-default values to the `initial_delay` and `multipier` attributes and checking the returned value).